### ISSUE TITLE
Powkiddy A13 screen resolution won't fit screen size

### ISSUE THREAD

**Issue by mikelandzelo173** [Ruka-CFW/rk3128-cfw#65]
**Describe the bug**
I have installed the Rev D firmware on my Powkiddy A13 and everything works fine so far. The only problem is, that the configuration file causes the UI to be kind of messed up. It won't fit the whole screen. It auto-detects `retroarch_v3.cfg` from your SD card structure zip-file from your repo with this configuration:

```
video_fullscreen_x = "800"
video_fullscreen_y = "480"
video_fullscreen = "true"
```

However, the screen resolution of the Powkiddy A13 is 1024x600. Changing the values to these dimensions and booting into the system causes the screen to turn on and off and on again before finally turning off one last time. Then it stays black. That's it. Changing these values to 800x480 again brings back the UI.

Games will also be affected by this, not only RetroArch.

Using `retroarch.cfg` from your zip-file which would have the correct resolution configured has the same effect.

I've also tried installing this firmware first as it was the only one labeled with "A13":
A12/A13 | A, B | RUKA 3.0 alpha CFW | RUKA_a12_v1_v2_3.0_alpha_20210504.img.zip
But after flashing the firmware the console wouldn't boot anymore and the screen just stayed black.

I'm not sure if this is a bug with the firmware or a problem on my end. Any help would be appreciated.

**Custom Firmware version:**
v3.0 Alpha 1

**Console (please complete the following information):**
 - Brand: Powkiddy
 - Model: A13
 - Version: The latest one, I guess. Bought it a month ago.
 - Mods: None

**To Reproduce**
1. Download "SDCard structure for RUKA v3.0 alpha" from your repo
2. Extract it on the SD card
3. Rename `retroarch v3.cfg` to `retroarch_v3.cfg` in `settings/retroarch/`
4. Put SD card into the device
5. Turn on the device
6. Screen is messed up but works
7. Exit RetroArch
8. Take out SD card and put it into your PC
9. Edit `retroarch_v3.cfg` and set correct screen resolution to 1024x600
10. Put the SD card back into the device
11. Turn on the device
12. You can see the (messed up) startup screen, then the screen flashes a few times until the screen finally turns off

**Expected behavior**
Video output that fits the screen at 1024x600 on the boot screen and the UI/games.

**Screenshots**
![IMG_20211215_085543372_HDR](https://user-images.githubusercontent.com/48515444/146150709-27b24499-1b18-4cde-b468-ecb36dcbcbff.jpeg)

**Comment by alpgarcia** [comment]
Why did you install Rev D for your a13? (Rev D is only for a12)

![Screenshot from 2021-12-15 11-16-20](https://user-images.githubusercontent.com/3502941/146168047-b5458556-b125-4e2f-a6e9-02ba5df1f2e2.png)

**Comment by fakemaria** [comment]
Closing, not an issue and already described on instructions the likelihood of firmwares to install.

**Comment by mikelandzelo173** [comment]
At first, I installed Rev A/B because it was the only CFW labeled as "A13". But after flashing the image the device did not boot and trying to flash another image resulted in the IDB error which was mentioned in the install guide. I had to take the device apart and press that other button combination to be able to flash a new firmware again.

So I was second-guessing myself because the install guide also mentions "Please install first Rev D firmware. If that didn´t work, install Rev A or B.". It does not state that this only applies to A12. So those text passages should probably be labeled as "A12 ONLY".

So I guess I'll try flashing A/B again? What if the device won't boot again?

**Comment by fakemaria** [comment]
> At first, I installed Rev A/B because it was the only CFW labeled as "A13". But after flashing the image the device did not boot and trying to flash another image resulted in the IDB error which was mentioned in the install guide. I had to take the device apart and press that other button combination to be able to flash a new firmware again.
> 
> So I was second-guessing myself because the install guide also mentions "Please install first Rev D firmware. If that didn´t work, install Rev A or B.". It does not state that this only applies to A12. So those text passages should probably be labeled as "A12 ONLY".
> 
> So I guess I'll try flashing A/B again? What if the device won't boot again?

Which version of board is your a13? As far as we know there is only rev a/b a13. So install that and make sure you have the proper files. You can stop by telegram and try to talk me there (same nick)

**Comment by derekho123** [comment]
Just want to report that I experienced exactly the same issue on my new A13. 

I bought my A13 two weeks ago and I believe it has a TN screen due to limited viewing angle. The default factory firmware is completely different from anything I have seen on YouTube. I found this Chinese video which shows this version:

[https://www.bilibili.com/video/BV1Uf4y1T7St](url)

Here is what happened to me:

(1) Flashed Ruka ver A/B and but device bricked and will not boot up. 
(2) Tried to flash Ruka verA/B one more time again but got "IBD fail" error.
(2) Flashed factory firmware using Android Tool but device still won't boot up.
(3) Opened the case and pressed the internal button, started device in MASKROM mode
(4) In RKDevTool, I click Erase LBA (or Erase All, can't remember)
(5) Flashed Ruka ver D, device now boot up but has messed up resolution
(6) Flashed factory firmware again using Android Tool, now device back to default and working again.

Anyway, seems like I might have a new hardware revision of the A13 that is not yet support by Ruka. I will greatly appreciate if this issue can be fixed.
